### ISSUE TITLE
Feature request: Support non-local restic repositories

### ISSUE THREAD

**Issue by stv0g** [triole/rsac#1]
As I understand, the tool only supports local restic repositories.

Do you plan to support remote storage backends such as S3 as well?

**Comment by triole** [comment]
Hi, yes that's right. The tool only supports checking the local file system for the latest backups. The problem I tried to solve was checking if the backups were up to date on the machine were they were saved without having repository access data. It is pretty simple because it basically looks up the latest files and their age without actually knowing what backups or data they contain. It is pretty simple and can also be done with a shell script I guess. 

If you want to check remote backups I suppose that you have the repository access data. In this case it might be easier to use something like [resticprofile](https://github.com/creativeprojects/resticprofile), run the `list` command to get a list of all backups and check their age by parsing the output with a shell script. At least that is what I would try first. To sum it up supporting remote repositories never was my intention because I wanted to solve the specific problem that I described above. Hope this helps.
